Ports/NiosII/GNU/os_cpu_c.c

```diff
@@ -0,0 +1,259 @@
+}
+
+
+/*
+*********************************************************************************************************
+*                                         TASK DELETION HOOK
+*
+* Description: This function is called when a task is deleted.
+*
+* Arguments  : ptcb   is a pointer to the task control block of the task being deleted.
+*
+* Note(s)    : 1) Interrupts are disabled during this call.
+*********************************************************************************************************
+*/
+
+void OSTaskDelHook (OS_TCB *ptcb)
+{
+    ptcb = ptcb;                                                /* Prevent compiler warning                             */
+}
+
+
+/*
+*********************************************************************************************************
+*                                          TASK SWITCH HOOK
+*
+* Description: This function is called when a task switch is performed.  This allows you to perform
+*              other operations during a context switch.
+*
+* Arguments  : none
+*
+* Note(s)    : 1) Interrupts are disabled during this call.
+*              2) It is assumed that the global pointer 'OSTCBHighRdy' points to the TCB of the task that
+*                 will be 'switched in' (i.e. the highest priority task) and, 'OSTCBCur' points to the
+*                 task being switched out (i.e. the preempted task).
+*********************************************************************************************************
+*/
+
+void OSTaskSwHook (void)
+{
+}
+
+
+/*
+*********************************************************************************************************
+*                                         STATISTIC TASK HOOK
+*
+* Description: This function is called every second by uC/OS-II's statistics task.  This allows your
+*              application to add functionality to the statistics task.
+*
+* Arguments  : none
+*********************************************************************************************************
+*/
+
+void OSTaskStatHook (void)
+{
+}
+
+
+/*
+*********************************************************************************************************
+*                                              TICK HOOK
+*
+* Description: This function is called every tick.
+*
+* Arguments  : none
+*
+* Note(s)    : 1) Interrupts may or may not be ENABLED during this call.
+*********************************************************************************************************
+*/
+
+/*
+ * Iniche stack has no header declaration for its timer 'hook'.
+ * Do that here to avoid build warnings.
+ */
+#ifdef ALT_INICHE
+void cticks_hook(void);
+#endif
+
+void OSTimeTickHook (void)
+{
+#if OS_TMR_EN > 0
+    OSTmrCtr++;
+    if (OSTmrCtr >= (OS_TICKS_PER_SEC / OS_TMR_CFG_TICKS_PER_SEC)) {
+        OSTmrCtr = 0;
+        OSTmrSignal();
+    }
+#endif
+
+#ifdef ALT_INICHE
+    /* Service the Interniche timer */
+    cticks_hook();
+#endif
+}
+
+void OSInitHookBegin(void)
```

Comment:
* Add 1 space before and after OSInitHookBegin
* do the same for lines 243, 247, 251 and  255